pyproject.toml

```diff
@@ -1,5 +1,38 @@
+[project]
+name = "oca_port"
+authors = [
+    {name = "Sébastien Alix", email="sebastien.alix@camptocamp.com"}
```

Comment:
Shall we have OCA too here as we do for modules?

Reply:
Added